ro_help/hub/models.py

```diff
@@ -197,6 +197,7 @@ class ResourceTag(TimeStampedModel):
     class Meta:
         verbose_name_plural = _("Resource tags")
         verbose_name = _("Resource tag")
+        ordering = ('name', )
```

Comment:
Hmmm...it looks like this needs to have a migration.

Reply:
fixed